src/sections/enterprise-offering/enterprise-bullets/enterprise-bullets.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { HTMLAttributes } from 'react';
+import classNames from 'classnames';
+import { Grid } from '@teambit/base-ui.layout.grid-component';
+import { Image } from '@teambit/evangelist.elements.image';
+import { text } from '@teambit/base-ui.layout.align';
+import { Paragraph, PossibleSizes } from '@teambit/base-ui.text.paragraph';
+import { H4 } from '@teambit/evangelist.elements.heading';
+import styles from './enterprise-bullets.module.scss';
+
+/**
+ * @name EnterpriseBullets
+ * @description
+ * A section showing few advantages of the Bit enterprise solution.
+ */
```

Comment:
why add this if you already have it in the `docs..md`?

Reply:
You right, not need this. I'll remove it.